src/gwt/src/org/rstudio/studio/client/workbench/views/packages/PackagesPane.java

```diff
@@ -344,6 +350,15 @@ public void run()
    
    private void initPackagesTable()
    {
+      Scheduler.get().scheduleDeferred(new ScheduledCommand()
+      {
+         @Override
+         public void execute()
+         {
+            onPackagesTableReady();
```

Comment:
Should this be done in `onAttach`? I'm not sure there's a guarantee of a live DOM here (although if it appears to work reliably that's probably good enough!)